api/test/build/go_build_test.go

```diff
@@ -3,19 +3,14 @@ package go_build_test
 import (
 	"testing"
 
-	_ "github.com/envoyproxy/data-plane-api/api/ads"
-	_ "github.com/envoyproxy/data-plane-api/api/als"
-	_ "github.com/envoyproxy/data-plane-api/api/bootstrap"
-	_ "github.com/envoyproxy/data-plane-api/api/cds"
-	_ "github.com/envoyproxy/data-plane-api/api/cert"
-	_ "github.com/envoyproxy/data-plane-api/api/eds"
-	_ "github.com/envoyproxy/data-plane-api/api/hds"
-	_ "github.com/envoyproxy/data-plane-api/api/lds"
-	_ "github.com/envoyproxy/data-plane-api/api/metrics_service"
-	_ "github.com/envoyproxy/data-plane-api/api/rds"
-	_ "github.com/envoyproxy/data-plane-api/api/rls"
-	_ "github.com/envoyproxy/data-plane-api/api/sds"
-	_ "github.com/envoyproxy/data-plane-api/api/trace_service"
+	_ "github.com/envoyproxy/data-plane-api/api/envoy/api/v2"
```

Comment:
We'll change this to go-control-plane once we have a branch there with regular protos.

Reply:
It looks it should be possible to generate gogo stuff as well. It's blocked on converting one more dependency to gogo so should be separated for now.